src/Countries/Bahrain.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+
+class Bahrain extends Country
+{
+    public function countryCode(): string
+    {
+        return 'bh';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'New Year Day' => '1-1',
+            'Labour Day' => '5-1',
```

Comment:
This PR lacks holidays. Is it possible to extend with all public holidays? For islamic holidays, you could look at how Turkey or Egypt did the implementation: https://github.com/spatie/holidays/pull/70